build/index.sh

```diff
@@ -19,38 +19,53 @@ echo '  downloading from ffmpeg.zeranoe.com'
 download 'https://ffmpeg.zeranoe.com/builds/win64/static/ffmpeg-latest-win64-static.zip' win32-x64.zip
 echo '  extracting'
 unzip -o -d ../bin/win32/x64 -j win32-x64.zip '**/ffmpeg.exe'
+echo '  compressing'
+$tar_exec -C ../bin/win32/x64 -czvf ../bin/win32/x64/ffmpeg.tar.gz ffmpeg.exe
```

Comment:
Out of curiosity: Why compress the executable before uploading? Modern web servers & CDNs should do this on-the-fly anyways, right?

Also, keep in mind that [the `tar` package adds another 7 dependencies](http://npm.broofa.com/?q=tar) just needed for *installing* `ffmpeg-static`.

Reply:
After an investigation it seems the compressed version is slower or about the same time to install (except for one likely fluke), so I will revert the compression change. https://app.circleci.com/github/jperl/test-ffmpeg-install-speed/pipelines